drivers/kscan/Kconfig.npcx

```diff
@@ -0,0 +1,69 @@
+# NPCX Keyboard Scan driver configuration options
+
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig KSCAN_NPCX
+	bool "Nuvoton NPCX embedded controller (EC) Keyboard Scan (KSCAN) driver"
+	default y
+	depends on DT_HAS_NUVOTON_NPCX_KSCAN_ENABLED
+	select MULTITHREADING
+	help
+	  This option enables the KSCAN driver for NPCX family of
+	  processors.
+
+if  KSCAN_NPCX
+
+config KSCAN_NPCX_COLUMN_SIZE
+	int "Keyscan NPCX Column Size"
+	default 18
```

Comment:
How about adding `range 1 18` here? Once the column size in defconfig file exceeds the range, we'll get an error message that indicates the configuration is out of range npcx ec can support.

Reply:
Done.

Reply:
Done.